python-package/lightgbm/basic.py

```diff
@@ -1853,6 +1853,119 @@ def free_network(self):
         self.network = False
         return self
 
+    def trees_to_dataframe(self):
+        """Parse the fitted model and return in an easy-to-read pandas DataFrame.
+
+        Returns
+        -------
+        DataFrame or list
+            Returns a pandas Dataframe if pandas is installed, and a list otherwise.
```

Comment:
```suggestion
            Returns a pandas DataFrame if pandas is installed, and a list otherwise.
```

Reply:
I will update this once we agree to what other changes are needed. :)